torch_geometric/transforms/line_graph.py

```diff
@@ -39,6 +39,12 @@ def __call__(self, data: Data) -> Data:
         (row, col), edge_attr = coalesce(edge_index, edge_attr, N, N)
 
         if self.force_directed or data.is_directed():
+# ==============change==================
+            print("It's a direct graph")
+            old_src, old_dst = data.edge_index #
+            line_graph_edge_attr = data.x[old_dst]#
+            (_,_),line_graph_edge_attr = coalesce(edge_index, line_graph_edge_attr, N, N)
+# ======================================
```

Comment:
The comments indicating parts of the code that changed can be remove, as github highlights changes automatically.
```suggestion
            old_src, old_dst = data.edge_index
            line_graph_edge_attr = data.x[old_dst]
            (_,_),line_graph_edge_attr = coalesce(edge_index, line_graph_edge_attr, N, N)
```